README.md

```diff
@@ -55,6 +56,9 @@ as our primary mechanism to receive code contributions.
 
 The Zulip community has a [Code of Conduct][code-of-conduct].
 
+[czo-doc]: https://zulip.readthedocs.io/en/latest/chat-zulip-org.html
+[zulip-devel]: https://groups.google.com/forum/#!forum/zulip-devel
+
```

Comment:
Done for `czo-doc`; the other is actually just here, as the other links point back up to this section. (These links aren't really parallel; one is a link to another doc with explanation, the other a direct link to the list with this section itself providing the explanation.)